openapi.yaml

```diff
@@ -488,6 +488,10 @@ components:
           maxLength: 128
         fields:
           $ref: "#/components/schemas/GrantFieldMap"
+        grantManagers:
+          type: array
+          items:
+            $ref: "#/components/schemas/PublicKeyMap"
```

Comment:
I think we only need wallet ID here -- not the full map

Reply:
We will still need the pubKey at that time to encrypt PII data.

Reply:
@adiwajshing removed map and just storing an array of address.